.travis.yml

```diff
@@ -1,4 +1,6 @@
 env:
+  - ACTIVE_CAMPAIGN_URL=https://account.api-us1.com/api/3
```

Comment:
Remind me again why this change is needed? Obviously it is 6am and I am still just waking up but I can't figure this one out :) 

Is it because of running the cassettes for merged pull requests?

Reply:
Hey @mhenrixon, I was trying to fix a error I received when running the pipeline in TravisCI: https://travis-ci.com/mhenrixon/active_campaign/jobs/291594389

But I'm not sure if it was the problem. Can you help me to find out this error in TravisCI?

Reply:
Shouldn't it be under `global:` then? The key is provided as a secret from travis in my branch.